doc/source/io.rst

```diff
@@ -4649,6 +4649,20 @@ destination DataFrame as well as a preferred column order as follows:
                              index_col='index_column_name',
                              col_order=['col1', 'col2', 'col3'], projectid)
 
+
+You can specify the query config as parameter
+
+.. code-block:: python
```

Comment:
say *why* this is useful as well. If you have a doc-link to things that you might want to pass here, pls add it.